src/apps/replication/lib/replica_learn.cpp

```diff
@@ -51,7 +51,7 @@ void replica::init_learn(uint64_t signature)
 
     if (status() != PS_POTENTIAL_SECONDARY)
     {
-        dwarn("%s: state is not potential secondary but %s, skip learning with signature %" PRIu64,
+        dwarn("%s: state is not potential secondary but %s, skip learning with signature[%016llx]",
```

Comment:
right, I mean PRIxxx actually supports hex format as well. No worry, we can do this in batch later.